.travis.yml

```diff
@@ -31,6 +31,7 @@ script:
   - if [ -n "${SHELLCHECK-}" ]; then shellcheck -s bash nvm.sh && shellcheck -s sh nvm.sh && shellcheck -s dash nvm.sh && shellcheck -s ksh nvm.sh ; fi
   - if [ -n "${SHELLCHECK-}" ]; then shellcheck -s bash install.sh bash_completion nvm-exec ; fi
   - if [ -n "${SHELL-}" ] && [ -n "${TEST_SUITE}" ]; then make TEST_SUITE=$TEST_SUITE URCHIN="$(npm bin)/urchin" test-$SHELL ; fi
+  - if [ -n "$WITHOUT_CURL" ]; then sudo apt-get install curl -y ; fi
```

Comment:
Okay, sure, but I have to say, there are more things mixed here so sometimes I really don't know which one would you prefer to use.